validator/validation/batches.py

```diff
@@ -128,7 +129,7 @@ def create_jobs(
     total_points = 0
 
     # if we've got data on a grid, process one cell at a time
-    if isinstance(reader, GriddedBase) or isinstance(reader, ReaderWithTsExtension):
+    if isinstance(reader, GriddedBase) or isinstance(reader, ReaderWithTsExtension) or isinstance(reader, SMAPL3_AM_PM):
```

Comment:
If the SMAP reader inherits from pynetcf (see comment below), this check should not be necessary